src/Dfc.CourseDirectory.Web/Controllers/ApprenticeshipsController.cs

```diff
@@ -881,10 +881,19 @@ public IActionResult DeleteDeliveryOption(DeleteDeliveryOptionViewModel model)
         [Authorize]
         public IActionResult AddNewVenue(ApprenticeshipLocationType type)
         {
-            _session.SetString("Option", type == ApprenticeshipLocationType.ClassroomBasedAndEmployerBased ?
-                "AddNewVenueForApprenticeshipsCombined" :"AddNewVenueForApprenticeships");
+            var returnUrl = type == ApprenticeshipLocationType.ClassroomBasedAndEmployerBased ?
+                Url.Action("DeliveryOptionsCombined", "Apprenticeships", new
+                {
+                    message = "",
```

Comment:
Not sure why we're using `message` to build the url, do we need to provide it, or just because legacy? If it's needed, use `string.Empty`.

Reply:
It was ported directly from legacy stuff, I believe. I wanted minimal changes in these legacy journeys.

Reply:
Sure, use `string.Empty`.